redcap_bridge/server_elab_interface.py

```diff
@@ -0,0 +1,153 @@
+import json
+import os
+
+import elabapi_python
+
+from redcap_bridge.utils import conversion_csv_to_json
+
+
+
+def download_experiment(server_config_json, id):
+
+    api_client = get_elab_config(server_config_json)
+    experiment_api = elabapi_python.ExperimentsApi(api_client)
+
+    exp = experiment_api.get_experiment(id)
+
+    return exp
+
+
+def create_template_with_metadata(server_config_json, template_file):
```

Comment:
Is there a reason this function is called `create_...` instead of `upload_...` like in the RedCap server interface?

Reply:
@killianrochet What do you think of this? Could you comment?

Reply:
and this?

Reply:
Sorry, didn't see this comment, i have changes name from create to upload it's better yes

Reply:
You are still using `create` in the function names of the corresponding tests. Can you also rename it there?